### ISSUE TITLE
Warning on metadata files not in manifest when they are

### ISSUE THREAD

**Issue by Aryllen** [Sage-Bionetworks/dccvalidator#351]
I'm getting a warning that I don't have metadata files in the manifest when I do. This might have to do with using Windows paths having backslashes. If I change the backslash to a forward slash, the problem goes away.

![image](https://user-images.githubusercontent.com/25048342/80027199-39552c00-8498-11ea-9c51-62c63f42c1eb.png)

![image](https://user-images.githubusercontent.com/25048342/80027246-50941980-8498-11ea-8232-d871fe4d092a.png)

**Comment by karawoo** [comment]
Yeah, I think it's an issue of recognizing file paths. `basename()` isn't finding the base name correctly. Reprex:
``` r
file <- "path\nC:\\Sage\\test1.csv\n/Sage/test2.csv"
dat <- readr::read_tsv(file)
check_files_manifest(dat, c("test1.csv", "test2.csv"))$data
#> [1] "test1.csv"
basename(dat$path)
#> [1] "C:\\Sage\\test1.csv" "test2.csv"
```

<sup>Created on 2020-04-22 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0.9001)</sup>
